jsk_2019_10_spatula/euslisp/pr2-spatula-force-test.l

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env roseus
+;;;;
+;;;; make PR2 use a spatula to scrape through a bowl without feedback, this is only a simple experiment in order to gather data for analysis of PR2 effort accuracy
+;;;;
+
+(load "package://pr2eus/pr2-interface.l")
+(ros::roseus "pr2_send_joints")
+;;set this flag to decicde whether to use simulation or real robot
+(setq simulation t)
+
+ 
+(if simulation
+    (progn
+	   (pr2)
+	   (when (not (boundp '*irtviewer*))
+	     (make-irtviewer)
+	     (objects (list *pr2*))
+	     (send *irtviewer* :draw-objects))
+	   
+	   (setq *ri* (instance pr2-interface :init))	   
+     )  
+    (pr2-init)  
+)
```

Comment:
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/pr2-interface.l#L630-L642
I think `(pr2-init :create-viewer t)` can be used for both real robot and simulated robot.